### ISSUE TITLE
Selecting file on mobile browser results in binded element being null instead of object

### ISSUE THREAD

**Issue by staxas** [danialfarid/ng-file-upload#1437]
When selecting a file on a desktop browser using an input element containing the ngf-select directive and 2-way bound to a "picture" element via ng-model, the resulting "picture" element contains an object which can be used to upload a file. However, when selecting a file in the same way on a mobile browser on an android device, the "picture" element contains null. On the android device, when clicking the input button, a dialog is opened which allows the user to select an app which will set the input to the input tag. The issue might be arising from this.

**Comment by wstam88** [comment]
+1

**Comment by danialfarid** [comment]
create a jsfiddle with steps to reproduce

**Comment by Medjeti** [comment]
Same problem here, running Chrome on Android. Everything works fine on desktop.
I can upload an image just fine, but if I select a video the file object always returns null. 

`
<button ngf-select="selectFile(uploadedFile)" required ng-model="uploadedFile" ngf-accept="'image/jpeg,video/*'" ngf-max-size="20MB">Select a file</button>
`

`
function selectFile(file) {
			if (file) {
				// Yay, we selected a file!
			} else {
				// No file selected :/
			}
		}
`

**Comment by danialfarid** [comment]
`selectFile($file)`

**Comment by Medjeti** [comment]
Hi Danial, thanks for your swift reply!

I changed the markup as per your suggestion, but the issue remains.

In the mean time I have succeeded in uploading a few clips - these where tiny little clips of a few seconds, so I'm thinking it might have to do with file size. It would seem that the Android app doesn't respect the ngf-max-size attribute but the script rejects the file once I return from the app.

I'll keep investigating - thanks for helping out.
